pandas/io/pickle.py

```diff
@@ -96,7 +97,9 @@ def try_read(path, encoding=None):
         # cpickle
         # GH 6899
         try:
-            return read_wrapper(lambda f: pkl.load(f))
+            with warnings.catch_warnings():
+                warnings.simplefilter('ignore')
```

Comment:
you don't need the simple filter

``warnings.catch_warnings(record=True)``

Reply:
Oh, I didn't know that.